config/v1/types_image_content_source_policy.go

```diff
@@ -0,0 +1,43 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ImageContentSourcePolicy holds cluster-wide information about how to handle registry mirror rules.  The canonical name is `cluster`
+type ImageContentSourcePolicy struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// spec holds user settable values for configuration
+	// +required
+	Spec ImageContentSourcePolicySpec `json:"spec"`
+}
+
+type ImageContentSourcePolicySpec struct {
+	// DigestMappingRules holds the registry mirroring rules that are written to
+	// /etc/containers/registries.conf to be used by the runtime to determine which
+	// repo to fall back on, if the original repo cannot be accessed.
+	// Will also be used for offline installs/upgrades.
```

Comment:
I think this should be changes to for example,

Reply:
Fixed